### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/range-interval#1]
Make a demo?

**Comment by ArtskydJ** [comment]
Have a few pre-filled input boxes with start/end/interval/step values, and a "Try Me!" button.

When they click the button, the button gets grayed out, and a box somewhere is populated with the `index` value each time `each` fires. When the `cb` fires, a box is populated with "end!", and the button is ungrayed. When pressed again, both boxes must be cleared.

**Comment by ArtskydJ** [comment]
```
+--------------------+    +---------+
| start    [_______] |    |         |
| end      [_______] |    | Try Me! |
| interval [_______] |    |         |
| step     [_______] |    |         |
+--------------------+    +---------+

+--------------------+    +---------+
|                    |    |         |
|        each        |    |   end!  |
|                    |    |         |
+--------------------+    +---------+
```

**Comment by ArtskydJ** [comment]
Fixed in 5b04acc5846940925b0770f57b11f57314b2a20a.
